### DISCUSSION TITLE
Is an AppImage available for download?

### DISCUSSION THREAD

**Discussion by probonopd** [brittonhayes/pod#7] [Q&A] [answered]
Is an AppImage available for download? If so, we'd like to add it to https://github.com/AppImage/appimage.github.io.
Thanks.

**Comment by brittonhayes** [comment]
Hey there @probonopd! Thanks for reaching out. I've just fixed the release action to include an `AppImage` of pod :slightly_smiling_face: 

I'll update this issue when I've confirmed that's working successfully.

**Comment by brittonhayes** [comment, marked as answer]
@probonopd As of [v0.7.6](https://github.com/brittonhayes/pod/releases/tag/v0.7.6), an AppImage is distributed in the application release. Would love to see pod added to [appimage.github.io](https://github.com/AppImage/appimage.github.io)! :heart: 

The application is still very early beta and does not contain all functionality, but I'm slowly chipping away at it. And nevertheless, it does now and will continue to fully support AppImage.

**Reply by probonopd** [reply]
Thank you very much @brittonhayes

**Comment by probonopd** [comment]
Thank you very much for providing an AppImage. Unfortunately, it was built on a "too new" system, so it doesn't run, e.g., on the still-supported Ubuntu bionic:

```
libm.so.6: version `GLIBC_2.29' not found
```

Could you please build the AppImage on Ubuntu bionic or earlier for better binary compatibility with older systems?

Test log:
https://github.com/AppImage/appimage.github.io/pull/2767/checks?check_run_id=3704121600#step:4:402

**Reply by brittonhayes** [reply]
Good callout! I've set the packaging process in Github actions to now use Ubuntu 18.04. Hopefully this will remediate the compatibility issues.

The release is building now so it will be available at [releases/latest](https://github.com/brittonhayes/pod/releases/latest) shortly.

Edit: https://github.com/brittonhayes/pod/releases/tag/v0.7.8

**Reply by probonopd** [reply]
Thank you very much @brittonhayes. Unfortunately it doesn't pass the automated test yet, but I think this should be solvable by bundling more dependencies: https://github.com/AppImage/appimage.github.io/pull/2767#issuecomment-927179486
